server/routes/api.py

```diff
@@ -55,5 +59,23 @@ def post(author_id):
         db.session.add(post)
         db.session.commit()
         return Response(status=httpStatus.OK)
+    
+@bp.route('/hello_world')
+def hello_world():
+    return make_response(jsonify(message='Hello World')), 200
+
+@bp.route('/verify_login')
+def verify_login():
+    # get token from authorization header
+    authorization_header = request.headers['Authorization']
+    token = authorization_header.replace("Bearer ", "")
 
+    try:
+        decoded_token = auth.verify_id_token(token)
+        uid = decoded_token['uid']
+        print("UID", uid)
+    except Exception as e:
+        print("Error", str(e))
+        return make_response(jsonify(message='There was an error' + str(e))), 500
```

Comment:
I keep getting a "Token used too early" error.

Reply:
This is odd, it works from my end and this is in the documentation. I'll take a look at it

Reply:
I manually synced my time with google servers as well and the issue still persisted.

Reply:
I updated this. You can try again, if it doesn't work, let me know and I'll take a look at it again

Reply:
I updated this. You can try again, if it doesn't work, let me know and I'll take a look at it again

Reply:
works now